ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/claimantresponse/ClaimantResponseMapper.java

```diff
@@ -67,7 +71,9 @@ private CCDClaimantResponse toRejection(Claim claim, ResponseRejection responseR
         responseRejection.getMediationContactPerson().ifPresent(rejection::mediationContactPerson);
         responseRejection.getMediationPhoneNumber()
             .ifPresent(phoneNo -> rejection.mediationPhoneNumber(telephoneMapper.to(phoneNo)));
-        responseRejection.getAmountPaid().ifPresent(rejection::amountPaid);
+
+        rejection.amountPaid(responseRejection.getAmountPaid().map(moneyMapper::to).orElse(null));
```

Comment:
this can be better written as `responseRejection.getAmountPaid().map(moneyMapper::to).ifPresent(rejection::amountPaid);`

Reply:
please change other places as well.

Reply:
Done